<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>tree.exe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>164</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Number of Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="pointNumber"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="biggerbutton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>放大</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="smallerbutton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>缩小</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cleanallbutton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Getpoint" name="getpoint" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load"/>
   </widget>
   <widget class="QMenu" name="menu_Graph">
    <property name="title">
     <string>&amp;Graph</string>
    </property>
    <addaction name="action_bigger"/>
    <addaction name="action_smaller"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Graph"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Load">
   <property name="text">
    <string>&amp;LoadFile</string>
   </property>
   <property name="toolTip">
    <string>LoadFile</string>
   </property>
  </action>
  <action name="action_bigger">
   <property name="text">
    <string>&amp;bigger</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="action_smaller">
   <property name="text">
    <string>&amp;smaller</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Getpoint</class>
   <extends>QWidget</extends>
   <header>getpoint.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>biggerbutton</sender>
   <signal>clicked()</signal>
   <receiver>action_bigger</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smallerbutton</sender>
   <signal>clicked()</signal>
   <receiver>action_smaller</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
